package.json

```diff
@@ -31,40 +31,32 @@
     "url": "https://github.com/netlify/cli/issues"
   },
   "scripts": {
-    "prepare": "is-ci || husky install node_modules/@netlify/eslint-config-node/.husky/",
-    "start": "node ./bin/run.js",
     "build": "tsc",
+    "certs": "openssl req -x509 -out localhost.crt -keyout localhost.key -newkey rsa:2048 -nodes -sha256 -subj \"/CN=localhost\" -extensions EXT -config certconf",
     "dev": "tsc --watch",
-    "test": "run-s format test:dev",
-    "format": "run-s format:check-fix:*",
-    "format:ci": "run-s format:check:*",
-    "format:check-fix:lint": "run-s format:check:lint format:fix:lint",
```

Comment:
bonus win: this was running eslint twice... fixing also checks, so it was redundant!